io.github.KangLin.SerialPortAssistant.json

```diff
@@ -0,0 +1,88 @@
+{
+    "id": "io.github.KangLin.SerialPortAssistant",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "SerialPortAssistant",
+    "modules": [
+        {
+            "name": "cmark",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/commonmark/cmark/archive/refs/tags/0.31.1.tar.gz",
+                    "sha256": "3da93db5469c30588cfeb283d9d62edfc6ded9eb0edc10a4f5bbfb7d722ea802"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DBUILD_TESTING=OFF"
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        {
+            "name": "RabbitCommon",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitCommon.git",
+                    "commit": "f2ccefec47d948191b6060e9b89f5e82bf1307fc"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ]
+        },
+        {
+            "name": "SerialPortAssistant",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/SerialPortAssistant.git",
+                    "commit": "4acd70b997daa8f136f8d1fccd43fc15bafedf78"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "build-options": {
+                "env": {
+                }
+            },
+            "post-install": [
+                "sed -i 's|Path=/app/bin|Path=/bin|' /app/share/applications/io.github.KangLin.SerialPortAssistant.desktop"
+            ],
+            "cleanup": [
+            ]
+        }
+    ],
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+    ],
+    "cleanup-commands": [
+    ],
+    "finish-args": [
```

Comment:
this should be at the top before modules.

Reply:
Based on the build logic,  I think this one makes more sense.

Reply:
no. based on the convention of every other. raise your hand if you review packages and maintain *at large*. The logic is modules are always last in the manifest.

Reply:
Yes. I am review packages and maintain at large.

Reply:
feel free to ignore the review, merging can wait too.

Reply:
I checked the documentation and didn't find such a convention. This is not a critical issue.